Prog/2018/Washington region feature/cost burden for 80% median income.sas

```diff
@@ -0,0 +1,84 @@
+/**************************************************************************
+ Program:  cost burden for 80% median income.sas
+ Library:  IPUMS
+ Project:  NeighborhoodInfo DC
+ Author:   Yipeng
+ Created:  7/2/2018
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Among [inner region] households with incomes below 80% of the area median, XX percent of renters and YY percent of owners have unaffordable cost burdens [30% of income].
+
+ Modifications:
+**************************************************************************/
+
+%include "L:\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( IPUMS )
+
+
+data allstates;
+	set ipums.acs_2012_16_dc ipums.Acs_2012_16_md ipums.Acs_2012_16_va ipums.Acs_2012_16_wv ;
+	if puma in (101,102,103,104,105,1001,1002,1003,1004,1005,1006,1007,1101,1102,1103,1104,1105,
+				1106,1107,1301,1302,10701,10702,10703,51255,59301,59302,59303,59304,59305,59306,59307,59308,59309) then innercounty = 1;
+run;
+
+data allstates2;
+set allstates;
+	if (numprec = 1 and hhincome <= 108600*0.8*0.7) or (numprec=2 and hhincome <= 108600*0.8*0.8) or  (numprec=3 and hhincome <= 108600*0.8*0.9)
```

Comment:
@maggiesu0725 This isn't the way you should identify 80% AMI. We have a variable we create in all of the IPUMS files called _hud_inc_, that correctly categorizes households into the HUD income groups based on both income and size. 

hud_inc= | Label
-- | -- 
1 | Extremely low (0-30% AMI)
2 | Very low (31-50%)
3 | Low (51-80%) 
4 | Middle (81-120%)
5 | High (over 120%)

Sorry, I should have pointed that out. Can you just substitute what you have to use _hud_inc_ instead? Thank you. cc: @rpitingolo